lib/appsignal/config.ex

```diff
@@ -36,13 +42,32 @@ defmodule Appsignal.Config do
       |> Map.merge(app_config)
       |> Map.merge(env_config)
 
+
     # Config is valid when we have a push api key
     config =
       config
       |> Map.put(:valid, !empty?(config[:push_api_key]))
 
     Application.put_env(:appsignal, :config, config)
 
+    if config[:valid] && !config[:request_headers] && !test?() do
+      require Logger
+
+      Logger.warn("""
+      The :request_headers config was not set in the AppSignal configuration, falling back to the default list. Please explicitly list response headers to send to AppSignal in config/appsignal.exs:
+
+        request_headers: ~w(
+      #{multiline_suggested_request_headers()}
+        )
+
+      Or set the APPSIGNAL_REQUEST_HEADERS environment variable:
+
+        $ export APPSIGNAL_REQUEST_HEADERS="#{single_line_suggested_request_headers()}"
+
+      Please check https://github.com/appsignal/appsignal-elixir/pull/336 for more information on this change.
+      """)
+    end
```

Comment:
This warning is also being printed during the installation now, which is kind of weird. I think it's missing the config for the installation task when it sends a demo sample.